cli/tests/pcluster/api/controllers/test_cluster_operations_controller.py

```diff
@@ -1033,6 +1033,28 @@ def test_incompatible_version(self, client, mocker):
                 [],
                 [{"failureCode": "ClusterCreationFailure", "failureReason": "Failed to create the cluster."}],
             ),
+            (
+                "CREATE_FAILED",
+                [
+                    [
+                        {
+                            "StackId": "fake-id",
+                            "StackName": "fake-name",
+                            "ResourceType": "AWS::CloudFormation::WaitCondition",
+                            "ResourceStatus": "CREATE_FAILED",
+                            "ResourceStatusReason": "Cluster has been set to PROTECTED mode "
```

Comment:
Could you put the real Status Reason here you see from cloudformation stack start with: WaitCondition received failed message:

Reply:
Fixed. Thanks!